docs/modules/ROOT/pages/index.adoc

```diff
@@ -904,11 +904,22 @@ Any content including, but not limited to the _<<_named_actor,named actor’s>>_
 
 The effectiveness of such attacks will necessarily be dependent on the language and other related development tooling in use for any given implementation. Beyond reminding implementors that parsing and validation errors are a likely attack surface, it is outside the scope of this specification to provide language-specific guidance.
 
-==== Homoglyph and typo-squatting attacks
+==== Homoglyph and typo-squatting attacks and name duplication
```

Comment:
"Name collisions"? - Duplication suggests putting it twice?

Reply:
Speaking of putting things twice, I just realized we have an existing "Name collisions" section. I've cross-referenced the two sections and removed this part of this header.